Formula/s/stellar-cli.rb

```diff
@@ -0,0 +1,32 @@
+class StellarCli < Formula
+  desc "Stellar command-line tool for interacting with the Stellar network"
+  homepage "https://developers.stellar.org"
+  url "https://github.com/stellar/stellar-cli.git",
+      tag:      "v21.3.0",
+      revision: "11746937e813346272faf68e70cbdbc1b0df780f"
+  license "Apache-2.0"
+  head "https://github.com/stellar/stellar-cli.git", branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "rust" => :build
+  depends_on "openssl@3"
+
+  on_linux do
+    depends_on "pkg-config" => :build
+  end
+
+  def install
+    chdir "cmd/stellar-cli" do
+      system "cargo", "install", "--bin=stellar", "--features=opt", *std_cargo_args
+    end
```

Comment:
```suggestion
    system "cargo", "install", "--bin=stellar", "--features=opt", *std_cargo_args(path: "cmd/stellar-cli")
```
Why do we need `--bin` here?

Reply:
The stellar-cli crate contains two binaries, a `soroban` binary and a `stellar` binary. Both binaries do the same thing. In the past the cli used to be called `soroban` and when the name was changed the old binary was kept around, including the `--bin` just makes sure only the `stellar` binary gets built and installed. If preferred I can remove the `--bin` and both binaries will be installed.